docs/2.0/integration/testng.adoc

```diff
@@ -167,23 +167,36 @@ They can: be parametrized, make checks, have nested steps, and create attachment
 
 In order to define steps in Java code, you need to annotate the respective methods with **@Step** annotation.
 When not specified, the step name is equal to the annotated method name converted to human-readable format.
-To define an explicit step name, pass your own **String** argument:
+
+Note that steps' mechanics was revised and now it supports smart fields' analysis.
```

Comment:
There are a `allure-java-migration` module that contains old annotations. So to migrate to a new version you can use this module and in this case all you need is change code that uses `Allure.LIFECYCLE`. 

https://github.com/allure-framework/allure-java/tree/master/allure-java-migration